src/components/uielements/input/Input.tsx

```diff
@@ -2,28 +2,22 @@ import React from 'react'
 
 import { InputProps } from 'antd/lib/input'
 
-import { InputWrapper, InputPasswordWrapper } from './Input.style'
-import { Color, InputType, InputSecurity, Size } from './types'
+import { InputWrapper } from './Input.style'
+import { Color, InputType, Size } from './types'
 
 interface Props extends InputProps {
   sizevalue?: Size
   color?: Color
   typevalue?: InputType
-  security?: InputSecurity
 }
 
 const Input: React.FC<Props> = ({
   color = 'primary',
   sizevalue = 'normal',
   typevalue = 'normal',
-  security = 'normal',
   ...rest
 }): JSX.Element => {
-  return security === 'password' ? (
-    <InputPasswordWrapper color={color} sizevalue={sizevalue} typevalue={typevalue} {...rest} />
-  ) : (
-    <InputWrapper color={color} sizevalue={sizevalue} typevalue={typevalue} {...rest} />
-  )
+  return <InputWrapper color={color} sizevalue={sizevalue} typevalue={typevalue} {...rest} />
```

Comment:
I still think we can get rid of `Input.tsx` component by using a `styled(Input)` component only (not tested):

```
# In `Input.styles.tsx`

import * as A from 'antd/lib/input'

type CustomInputProps {
  sizevalue?: Size
  color?: Color
  typevalue?: InputType
}

export type InputProps = CustomInputProps & A.InputProps

export const Input = styled(A.Input)<InputProps>`
 //  ... add all styles here
`
```


```
// in any other component which needs the `Input` component

import * as Styled from './Input.styled'

<Styled.Input />

Reply:
yes. we can. the only thing here is that here are default values for 3 props. this component is used in 15 files and everywhere we should set this props manually 'cause setting defaultProps for styled component does not work with TS now.

Reply:
Fixed with b6bbb8d135cffd61aab07a9560c66ee04c629085